CHANGELOG.md

```diff
@@ -66,12 +66,22 @@ Swift Next
       // This will now error, because the protocol requirement
       // is implicitly mutating and the setter is implicitly 
       // nonmutating.
-      set { someProperty = newValue }
+      set { someProperty = newValue } // Error
     }
   }
   ```
 
-  To resolve this, explicitly mark the setter as `nonmutating` in the protocol.
+  **Workaround**: Define a new mutable property inside the setter that has a reference to `self`:
```

Comment:
Nitpick: "mutable variable" is probably easier for people to understand.

Reply:
Fixed!